CorsixTH/Lua/staff_profile.lua

```diff
@@ -142,12 +145,17 @@ function StaffProfile:randomise(month)
   self:randomiseOrganical()
 end
 
+--! Give the staff member some personal properties.
 function StaffProfile:randomiseOrganical()
-  -- TODO: Randomise a letter from the current language's alphabet.
-  self.name = "" -- string.char(string.byte"A" + math.random(0, 25)) .. ". "
+  local initials = staff_initials_cache.initials
+  local num = self.name_seed % #initials
+  self.initial = initials[num]
+  self.name = ""
   for _, part_table in ipairs(name_parts) do
     self.name = self.name .. part_table.__random
   end
+  print(self.initial, self.name)
```

Comment:
Is this intentional or a leftover print? Hard to tell on mobile

Reply:
Thanks for the catch.